_episodes/04-recipe.md

```diff
@@ -194,13 +202,14 @@ Let's dissect what's happening here.
 > > ## Answers
 > >
 > > 1. The config file should be the one we edited in the previous episode,
-> >    something like `/home/<user>/.esmvaltool/config-user.yml`.
+> >    something like `/home/<username>/.esmvaltool/config-user.yml` or `./config-user.yml`.
 > > 1. ESMValTool found the recipe in its installation directory, something like
-> >    `/home/user/miniconda3/envs/esmvaltool_tutorial/bin/esmvaltool/recipes/examples/`.
+> >    `/home/user/username/miniconda3/envs/esmvaltool/bin/esmvaltool/recipes/examples/`
```

Comment:
Both? `user/username`?

Reply:
@LisaBock - Yes, that's the path on JASMIN. Everyone's home dir is under /home/user/.

Reply:
Sorry..this is /home/users/username. I ahve fixed this now.